modules/nf-core/samtools/faidx/main.nf

```diff
@@ -9,11 +9,13 @@ process SAMTOOLS_FAIDX {
 
     input:
     tuple val(meta), path(fasta)
+    tuple val(meta2), path(fai)
```

Comment:
Why would we need fai as input when it's what we want to output?

Reply:
When you give it both fasta and fai, you can extract specific sequences as well.